Aliucord/src/main/java/com/aliucord/updater/PluginUpdater.java

```diff
@@ -170,6 +174,10 @@ public static boolean update(String plugin) throws Throwable {
             );
         }
 
+        try {
+            p.onAfterUpdate(p.getManifest().version);
```

Comment:
i think this should be called after the remount so it uses the method of the newly updated plugin instead of the old one

Reply:
Issue is that it might not always be triggered then (for example when the plugin is disabled when updated)